### ISSUE TITLE
flow-node breaks multiline statements

### ISSUE THREAD

**Issue by ianp** [leebyron/flow-remove-types#10]
The following input works with regular node, but `flow-node` tries to eval it as soon as it sees the first newline:

```js
const client = restify.createJsonClient({
  url: 'https://api.just-eat.com'
})
```

**Comment by ianp** [comment]
Ah, looks like it's fixed in 1.0.3, I was running 1.0.2. That'll teach me to open the issue before I start working on a fix 😁

**Comment by leebyron** [comment]
Thanks for your diligence none the less! Glad you're enjoying flow-node :)
